docs/how-to/restore-linstor-db.md

```diff
@@ -0,0 +1,87 @@
+# How to Restore a LINSTOR Database Backup
+
+This guide shows you how to restore a LINSTOR® Controller from a database backup.
```

Comment:
It ought to be obvious, but warn that changes made since the backup was taken are lost? I guess most things will be re-created by the CSI driver. How reliably are we able to pick up the existing volumes with their data? Should normally work, I suppose.